addons/website/static/src/components/views/theme_preview.xml

```diff
@@ -1,30 +1,45 @@
+                <i class="oi oi-arrow-left"/>
+            </button>
+        </xpath>
+        <xpath expr="//*[hasclass('o_control_panel_actions')]" position="attributes">
+            <attribute name="class" remove="justify-content-lg-around" add="justify-content-sm-end flex-grow-1 flex-shrink-1 flex-basis-sm-0" separator=" "/>
+        </xpath>
         <xpath expr="//div[hasclass('o_control_panel_navigation')]" position="replace">
-        <div class="d-none d-md-flex align-items-end justify-content-end order-2 order-lg-1 w-100 gap-1" role="group">
-                <input class="btn-check" id="themeViewerDesktop" type="radio" name="viewer" data-mode='desktop' autocomplete="off" checked='checked'/>
-                <label class="btn btn-secondary" for="themeViewerDesktop" t-on-click="this.onDesktopClick">Desktop</label>
-                <input class="btn-check"  id="themeViewerMobile" type="radio" name="viewer" data-mode='mobile' autocomplete="off"/>
-                <label class="btn btn-secondary" for="themeViewerMobile" t-on-click="this.onMobileClick">Mobile</label>
+            <div class="o_control_panel_switch_device d-none d-lg-flex" role="group">
+                <div class="btn-group">
+                    <input class="btn-check" id="themeViewerDesktop" type="radio" name="viewer" data-mode='desktop' autocomplete="off" checked='checked'/>
+                    <label class="btn btn-secondary" for="themeViewerDesktop" t-on-click="this.onDesktopClick">
+                        <i class="fa fa-desktop me-1"/>Desktop
+                    </label>
+                    <input class="btn-check"  id="themeViewerMobile" type="radio" name="viewer" data-mode='mobile' autocomplete="off"/>
+                    <label class="btn btn-secondary" for="themeViewerMobile" t-on-click="this.onMobileClick">
+                        <i class="fa fa-mobile me-1"/>Mobile
+                    </label>
+                </div>
             </div>
         </xpath>
     </t>
 
     <t t-name="website.ThemePreviewFormController" t-inherit="web.FormView" t-inherit-mode="primary">
         <xpath expr="//Layout" position="inside">
             <t t-set-slot="layout-actions">
-                <div>
-                    <div class="o_form_buttons_view" role="toolbar" aria-label="Main actions">
-                        <ViewButton className="'btn btn-primary o_use_theme'"
-                                    clickParams="{type: 'object', name: 'button_choose_theme'}"
-                                    record="this.model.root">
-                            <t t-set-slot="contents">Start Now</t>
-                        </ViewButton>
-                        <button class="btn btn-link o_switch_theme" t-on-click.stop="this.back">
-                            Choose another theme
-                        </button>
-                    </div>
+                <div class="o_form_buttons_view d-flex flex-row-reverse flex-lg-row" role="toolbar" aria-label="Main actions">
```

Comment:
Since you removed the second button, I think you can remove the parent `<div>` container since we do not need the flex properties anymore